### PR TITLE
sunxi: Fix WiFi on Sinovoip Banana Pi M2 Ultra

### PR DESCRIPTION
**Pull request by halmartin**
This enables onboard WiFi (AP6212) out of the box

Suggestion from @juanesf. Tested on trunk, onboard WiFi is now working.

Signed-off-by: Hal Martin <hal.martin@gmail.com>

### CODE REVIEW

**Comment by adrianschmutzler** [comment]
Please separate name change and package changes into two separate commits (inside this PR).

File: target/linux/sunxi/image/cortex-a7.mk [hunk 1]
```diff
@@ -63,13 +63,14 @@ define Device/lemaker_bananapi
 endef
 TARGET_DEVICES += lemaker_bananapi
 
-define Device/lemaker_bananapi-m2-ultra
-  DEVICE_VENDOR := LeMaker
+define Device/sinovoip_bananapi-m2-ultra
+  DEVICE_VENDOR := Sinovoip
   DEVICE_MODEL := Banana Pi M2 Ultra
-  DEVICE_PACKAGES:=kmod-rtc-sunxi kmod-ata-sunxi
+  DEVICE_PACKAGES := kmod-rtc-sunxi kmod-ata-sunxi kmod-brcmfmac \
```
Comment (adrianschmutzler) [inline target/linux/sunxi/image/cortex-a7.mk:69]: Please don't introduce spaces next to the equal sign, as this does not match the style of the remaining entries.

File: target/linux/sunxi/image/cortex-a7.mk [hunk 2]
```diff
@@ -63,13 +63,14 @@ define Device/lemaker_bananapi
...
+	 brcmfmac-firmware-43430a0-sdio wpad-basic
```
Comment (adrianschmutzler) [inline target/linux/sunxi/image/cortex-a7.mk:70]: Indent with one tab, no space.

**Comment by adrianschmutzler** [comment]
If this target uses sysupgrade, the name change will also require SUPPORTED_DEVICES to be extended.

**Comment by halmartin** [comment]
> If this target uses sysupgrade, the name change will also require SUPPORTED_DEVICES to be extended.

Only sdcard images are available, so I don't think this applies?

**Comment by adrianschmutzler** [comment]
Your commits have formal issues concerning their commit titles (prefix, length).
Despite, please add a justification/evidence for your second commit.
In the first commit, you should leave the vendor name as it is.

**Comment by halmartin** [comment]
I made an error when the device was added to OpenWrt, LeMaker was the manufacturer of the original Banana Pi but the Banana Pi M2 Ultra is manufactured by Sinovoip:
http://www.banana-pi.org/m2u.html (see page footer)

I have amended the commits as requested.

**Comment by hauke** [comment]
Thank you for the patches, they were applied to master.

**Pull request closed without merging**